tokenserver/scripts/purge_old_records.py

```diff
@@ -53,16 +53,21 @@ def purge_old_records(config_file, grace_period=-1, max_per_loop=10,
             logger.debug("Purging old user records for service: %s", service)
             # Process batches of <max_per_loop> items, until we run out.
             while True:
+                offset = random.randint(0, max_offset)
```

Comment:
In 3 months time I'm not going to remember why we do this; could you please add a short paragraph to the docstring of this function, along the lines of what you explained in the PR here?